packages/fast-usdc-contract/src/exos/settler.ts

```diff
@@ -438,7 +438,8 @@ export const prepareSettler = (
               reference === currentChainReference
                 ? E(settlementAccount).send(dest, amt)
                 : E(settlementAccount).transfer(dest, amt, {
-                    forwardOpts: { intermediateRecipient },
+                    timeoutRelativeSeconds: 10n * 60n,
```

Comment:
consider a module-level constant

Reply:
seems pretty clear and concise as is

Reply:
the reason to make a named constant is so that it's apparent that it is a design-time configuration option and how it compares semantically to other values that happen to be the same.

Reply:
to wit, the PR now has a second instance of this value. Are they meant to be kept in sync? If so please name a constant and use it in both places.

Reply:
ah; ok; I added `FORWARD_TIMEOUT`.

I thought you meant `const SEC_PER_MIN = 60n;` which doesn't seem to carry its weight.